source/Irrlicht/CBAWMeshFileLoader.h

```diff
@@ -52,7 +54,9 @@ class CBAWMeshFileLoader : public IMeshLoader
 
 private:
 	template<typename T>
-	T* make(SPair&, SContext&) const;
+	T* make(SBlobData&, SContext&) const;
+
+	void registerObject(SBlobData& _data, SContext& _ctx, IReferenceCounted* _obj) const;
```

Comment:
If you ditch `std::map<uint64_t, IReferenceCounted*> createdObjects` then you can most probably ditch this function too (or not, but it will just be almost a macro for a 1 line op).